scripts/zones/Throne_Room/Zone.lua

```diff
@@ -63,4 +63,8 @@ end;
 function onEventFinish(player,csid,option)    
     --printf("CSID: %u",csid);
     --printf("RESULT: %u",option);
+    if (csid == 0x06) then
+        -- You will be transported back to the entrance of Castle Zvahl Baileys
+        player:setPos(378.222,-12,-20.299,125,0xA1);
```

Comment:
I just tried and the CS moves to the entrance of Throne room.  It won't zone you at the entrance of Castle Zvahl Baileys.  I don't CS can zone you, only move you.  According to original script, we want to zone.